tests/testthat/test-as-data-frame.R

```diff
@@ -94,19 +95,21 @@ with_mock_crunch({
     test_that("csvToDataFrame handles hidden variables", {
         new_ds <- loadDataset("test ds")[, c("birthyr", "gender", "location", "mymrset", "textVar", "starttime")]
         new_ds$birthyr@tuple[["discarded"]] <- TRUE
-        new_ds_df <- as.data.frame(new_ds)
-        expect_silent(
+        new_ds_df <- as.data.frame(new_ds, include.hidden = FALSE)
+        expect_warning(
             expect_equal(names(csvToDataFrame(csv_df, new_ds_df)),
-                c("gender", "location", "subvar2", "subvar1", "subvar3", "textVar",
-                    "starttime"))
+                c("birthyr", "gender", "location", "subvar2", "subvar1", "subvar3", "textVar",
+                    "starttime"),
+                "Variable birthyr is hidden"),
+
         )
         # now we want the hidden vars to be included
         new_ds_df <- as.data.frame(new_ds, include.hidden = TRUE)
         expect_warning(
             expect_equal(names(csvToDataFrame(csv_df, new_ds_df)),
-                c("birthyr", "gender", "location", "subvar2", "subvar1", "subvar3", "textVar",
-                    "starttime")),
-            paste0("Variable birthyr is hidden"))
+                         c("birthyr", "gender", "location", "subvar2", "subvar1", "subvar3", "textVar",
+                           "starttime")),
+            "Variable birthyr is hidden")
     })
```

Comment:
Could we also have a test here showing what happens when `new_ds_df <- as.data.frame(new_ds); (names(csvToDataFrame(csv_df, new_ds_df)) ...`? I might be missing this test somewhere else, but if not we should include it to cover all of our bases

Reply:
Now that I think about it I think we can just remove this test entirely. Since we always have hidden variables in the `names(cdf)` `csvToDataFrame` is going to do the same thing regardless of what is passed to include.hidden.